Makefile

```diff
@@ -152,7 +152,6 @@ vendor: composer.lock
 
 vendor/bin/phpunit: composer.lock
 	composer install
-	composer require --dev phpunit/phpunit ^7.5
```

Comment:
This looks like it has been accidentally deleted. Maybe this is why unit tests are failing?